etcdctl/ctlv3/command/member_command.go

```diff
@@ -100,6 +101,20 @@ The items in the lists are ID, Status, Name, Peer Addrs, Client Addrs, Is Learne
 	return cc
 }
 
+// NewMemberPromoteCommand returns the cobra command for "member promote".
+func NewMemberPromoteCommand() *cobra.Command {
+	cc := &cobra.Command{
+		Use:   "promote <memberID>",
+		Short: "Promotes a non-voting member in the cluster",
+		Long: `Promotes a non-voting learner member to a voting one in the cluster.
+`,
```

Comment:
why a new line here?

Reply:
I did not write this part of the code. But it seems consistent with the existing `NewMemberListCommand`.